solutions/instances/variables.tf

```diff
@@ -43,6 +43,151 @@ variable "prefix" {
   default     = null
 }
 
+##############################################################################
+# IBM Cloud Logs
+##############################################################################
+variable "cloud_logs_provision" {
+  description = "Provision a IBM Cloud Logs instance?"
```

Comment:
Please refer the similar variables description in this DA and review all descriptions. Exisiting ones are already reviewed by Allen.